cowprotocol/token-lists
# Bumped version to 2.4.0

# Summary

Adding WETH on Gnosis chain
Plus updates to addToken form which are already live (once it hits develop they are active), no impact on main

# Testing

1. Add the list to cowswap and verify it loads https://raw.githubusercontent.com/cowprotocol/token-lists/814ddef95ff3c62ba8901038fc37db3d982fbc5f/src/public/CowSwap.json
2. Check it also on tokenlists: https://tokenlists.org/token-list?url=https://raw.githubusercontent.com/cowprotocol/token-lists/814ddef95ff3c62ba8901038fc37db3d982fbc5f/src/public/CowSwap.json
